src/Pester.Runtime.ps1

```diff
@@ -58,6 +58,59 @@ $flags = [System.Reflection.BindingFlags]'Instance,NonPublic'
 $script:SessionStateInternalProperty = [System.Management.Automation.SessionState].GetProperty('Internal', $flags)
 $script:ScriptBlockSessionStateInternalProperty = [System.Management.Automation.ScriptBlock].GetProperty('SessionStateInternal', $flags)
 $script:ScriptBlockSessionStateProperty = [System.Management.Automation.ScriptBlock].GetProperty("SessionState", $flags)
+# function for extracting StacktraceSystemLanguage
+function Get-StackTraceLanguageFallBack {
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$StackTrace
+    )
+    $StackTraceLanguage = [PSCustomObject]@{
+        At   = "at"
+        Line = "line"
+    }
+    $Regex = "(?<At>.*)\s(?<ScriptBlockOrFunction>\<\w+\>),\s(?<FileName>\<.+\>)\s*:\s(?<Line>\w+)\s(?<LineNumber>\w+)\z"
```

Comment:
```suggestion
    $Regex = "(?<At>.*)\s(?<ScriptBlock>\<\w+\>),\s(?<NoFile>\<.+\>)\s*:\s(?<Line>.+)\s(?<LineNumber>\d+)\z"
```
Not sure if "line" could be multiple words, but regex should be consistent between this and `Get-StackTraceLanguage`.

Also consider comment to make it clear that this only matches final line in stacktrace with scriptblock-source, e.g. `at <ScriptBlock>, <No file>: line 1`